packages/gestalt/src/Spinner/VRSpinner.css

```diff
@@ -0,0 +1,111 @@
+.spinner {
+  --g-dot-size: calc(var(--g-size) / 3);
+  --g-dot-space: calc(var(--g-dot-size) * 1.75);
+
+  height: var(--g-size);
+  width: var(--g-size);
+}
+
+.lg {
+  --g-size: var(--comp-spinner-size-lg);
+}
+
+.md {
+  --g-size: var(--comp-spinner-size-md);
+}
+
+.sm {
+  --g-size: var(--comp-spinner-size-sm);
+}
+
+.spinnerFrame {
+  animation: spin 1500ms linear infinite;
+  box-sizing: border-box;
+  height: calc(var(--g-dot-space) * sqrt(3) / 2);
+  left: calc(var(--g-dot-size) * 0.625);
+  position: relative;
+  top: calc(var(--g-dot-size) / 2);
+  transform-origin: center calc(var(--g-dot-space) / sqrt(3));
+  width: var(--g-dot-space);
+}
+
+.spinnerFrame > div {
+  animation-composition: add;
+  animation-duration: 800ms, 1.8s;
+  animation-iteration-count: infinite;
+  animation-name: scale, colors;
+  animation-timing-function: linear;
+  border-radius: 50%;
+  height: var(--g-dot-size);
+  position: absolute;
+  width: var(--g-dot-size);
+}
+
+.dot1 {
+  animation-delay: 0s;
+  left: 50%;
+  top: 0;
+  transform: translate3d(-50%, -50%, 0);
+}
+
+.dot2 {
+  animation-delay: 100ms, 0s;
```

Comment:
should this be a motion token?

Reply:
the same reason as [here](https://github.com/pinterest/gestalt/pull/3774/files#r1781101993).

I see these values as private.

Reply:
We were thinking about adding delay tokens but they don't appear as much. Maybe further down the line we can consider if there're many delayed animations, or some kind of patterns grow.